.github/issue_template.md

```diff
@@ -1,12 +1,34 @@
+> Below you will find NVDA's Github issue template. You might consider swapping to the preview tab in order
```

Comment:
I don't understand why this is in the diff, could you please merge most recent master?